mozilla-mobile/fenix/app/src/main/res/values/strings.xml

```diff
@@ -270,7 +270,7 @@
     <!-- Home onboarding dialog welcome screen description text. -->
     <string name="onboarding_home_welcome_description">More colors. Better privacy. Same commitment to people over profits.</string>
     <!-- Home onboarding dialog sign into sync screen title text. -->
-    <string name="onboarding_home_sync_title">Switching screens is easier than ever</string>
+    <string name="onboarding_home_sync_title">Hop from phone to laptop and back</string>
```

Comment:
This needs a new ID

Reply:
While adding a new ID, I was wondering if we could also completely remove the existing `onboarding_home_sync_title` since it was only added last week?

Reply:
Spoke with @flodolo, we can remove the old string since it's not used.